chainntnfs/test_utils.go

```diff
@@ -241,20 +243,20 @@ func NewBitcoindBackend(t *testing.T, minerAddr string, txindex,
 	}
 
 	if rpcpolling {
+		cfg.PollingConfig = &chain.PollingConfig{
```

Comment:
Nice catch! Can't believe this is wrong the whole time🤦🏻

Reply:
yeah - my fault! 🙈